### ISSUE TITLE
1.3.0-99: Windows: "Pipeline failed: source" on 2nd+ run

### ISSUE THREAD

**Issue by AirQuick** [ndw/xmlcalabash1#322]
Depending on `.xmlresolver` dir, 1.3.0-99 fails with `ERROR: Pipeline failed: source` on Windows 10. Not repro on Ubuntu 18.04.
The error has not happened on 1.2.5-99.

1. Clone XSpec v2.1.4 and `cd` to it:

   ```console
   PS C:\test> git clone -b v2.1.4 https://github.com/xspec/xspec.git
   Cloning into 'xspec'...
   (...snip...)
   Note: switching to 'd8bbeca51eaccc8b3e07c82547f6aff4a2072aef'.
   (...snip...)
   PS C:\test> cd xspec
   ```

1. Make sure you don't have `.xmlresolver` dir in your user profile dir.

1. Run `src\harnesses\saxon\saxon-xquery-harness.xproc` (`-p xspec-home=` must point the directory URI where XSpec is cloned). It works:

   ```console
   PS C:\test\xspec> java -jar ..\xmlcalabash-1.3.0-99\xmlcalabash-1.3.0-99.jar -i source=tutorial/xquery-tutorial.xspec -p xspec-home=file:///c:/test/xspec/ src\harnesses\saxon\saxon-xquery-harness.xproc
   INFO : src/harnesses/harness-lib.xpl:227:31:
   INFO : src/harnesses/harness-lib.xpl:291:45:passed: 1 / pending: 0 / failed: 0 / total: 1
   <?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   (...snip...)
   ```

1. Now you have `.xmlresolver` dir in your user profile dir.

1. Run `src\harnesses\saxon\saxon-xquery-harness.xproc` again with the exact same command line. This time it doesn't work:

   ```console
   PS C:\test\xspec> java -jar ..\xmlcalabash-1.3.0-99\xmlcalabash-1.3.0-99.jar -i source=tutorial/xquery-tutorial.xspec -p xspec-home=file:///c:/test/xspec/ src\harnesses\saxon\saxon-xquery-harness.xproc
   INFO : src/harnesses/harness-lib.xpl:227:31:
   ERROR: Pipeline failed: source
   ```

1. Remove `.xmlresolver` dir from your user profile dir.

1. Run `src\harnesses\saxon\saxon-xquery-harness.xproc` with the exact same command line. It works again.

**Comment by ndw** [comment]
Thanks, @AirQuick .  I've reproduced it on Windows. I'm at a loss to explain why it's Windows specific, but I've had "get my Windows dev environment functioning" on my todo list for a couple of weeks.

The bug is in `xmlresolver-2.1.0.jar`, so a quick workaround is to replace that jar in the `lib` directory with the 1.0.4 version from XML Calabash 1.2.5 or otherwise make sure that the 1.0.4 version comes first on your classpath.

Apologies for the inconvenience.

**Comment by fgeorges** [comment]
The backslashes look like usual suspects to me, here ;-)

**Comment by ndw** [comment]
Out of curiousity, @AirQuick what shell are you using on Windows? If it's not 'Command Prompt' can you try it with that shell and see if you get different results?

**Comment by AirQuick** [comment]
https://github.com/ndw/xmlcalabash1/issues/322#issue-919716666 was on PowerShell, but it's reproduced also on Command Prompt.

**Comment by ndw** [comment]
Fixed in 1.3.1.

**Comment by AirQuick** [comment]
Verified the fix. Thanks!
